webxrlayers-1.bs

```diff
@@ -240,18 +240,22 @@ The {{XRLayerLayout}} enum defines the layout of the layer.
 <pre class="idl">
 enum XRLayerLayout {
   "mono",
-  "stereo",
+  "default",
```

Comment:
This feels fine to me, although as the most minor nit ever: If we're gonna call it "default" then we should probably make it the first item in the enum. :)

Reply:
ok :-)